addons/prometheus/0.35.x/prometheus-2.yaml

```diff
@@ -42,7 +42,7 @@ spec:
   chartReference:
     chart: prometheus-operator
     repo: https://mesosphere.github.io/charts/staging
-    version: 8.8.2
+    version: 8.8.3
```

Comment:
Please also change `catalog.kubeaddons.mesosphere.io/addon-revision` to `0.35.0-2`